scopes/lanes/lanes/lane.cmd.ts

```diff
@@ -254,43 +254,62 @@ it is useful e.g. when having multiple lanes with the same name, but with differ
 }
 
 export class LaneChangeScopeCmd implements Command {
-  name = 'change-scope <remote-scope-name>';
+  name = 'change-scope <lane-name-or-remote-scope> <remote-scope-name>';
   description = `changes the remote scope of a lane`;
-  extendedDescription = 'NOTE: available only before the lane is exported to the remote';
+  extendedDescription = `If no lane is specified (i.e. only one argument provided) this will update the current lane.
+NOTE: available only before the lane is exported to the remote`;
   alias = '';
-  options = [
-    [
-      'l',
-      'lane-name <lane-name>',
-      'the name of the lane to change its remote scope. if not specified, the current lane is used',
-    ],
-  ] as CommandOptions;
+  options = [] as CommandOptions;
   loader = true;
   migration = true;
+  example: [
+    {
+      cmd: 'lane change-scope my-org.other-scope';
+      description: 'change the scope of the current lane to my-org.other-scope';
+    },
+    {
+      cmd: 'lane change-scope my-specific-lane my-org.other-scope';
+      description: "change the scope of lane 'my-specific-lane' to my-org.other-scope";
+    }
+  ];
 
   constructor(private lanes: LanesMain) {}
 
-  async report([remoteScope]: [string], { laneName }: { laneName?: string }): Promise<string> {
-    const { remoteScopeBefore, localName } = await this.lanes.changeScope(remoteScope, laneName);
+  async report([laneNameOrRemoteScope, remoteScope]: [string, string]): Promise<string> {
```

Comment:
Again this is confusing Please don't do it like this (with the or)
we should assign the destructed value with a default value
```
async report([laneName='currentLane', remoteScope]: [string, string]): Promise<string> {

Reply:
this wont work. If you provide a single argument it will be just the remoteScope, so the first argument isn't always the lane name

Reply:
this for me is much more confusing. Now it looks like the remoteScope is getting data from a lane.
I like the logic, but not the destructured variable naming, should keep the name as i put it and i'll change to this logic

Reply:
@NitsanCohen770 FYI this doesnt remove the need for the tertiary, just the makes the first line of the function redundant, which is fine by me.
I dont want to add unneccessary logic to the changeScope function, it works fine as it is